crypto/rand/rand_lib.c

```diff
@@ -581,6 +585,29 @@ size_t rand_pool_bytes_remaining(RAND_POOL *pool)
     return pool->max_len - pool->len;
 }
 
+static int rand_pool_grow(RAND_POOL *pool, size_t len)
+{
+    if (len > pool->alloc_len - pool->len) {
+        unsigned char *p;
+        const size_t limit = pool->max_len / 2;
+        size_t newlen = pool->alloc_len;
+
+        do
```

Comment:
braces here "do {" ? (not definitley sure about the style in this case)

Reply:
It isn't mentioned explicitly.  The most relevant seems to be: _do not unnecessarily use braces around a single statement_ which is talking about if statements but we do the same for some others.

Reply:
Okay.